blessed/sequences.py

```diff
@@ -15,7 +16,10 @@
 import wcwidth
 import six
 
-__all__ = ('init_sequence_patterns', 'Sequence', 'SequenceTextWrapper',)
+__all__ = ('init_sequence_patterns', 'Sequence', 'SequenceTextWrapper', 'iter_parse')
+
+
+TextFragment = collections.namedtuple('TextFragment', ['ucs', 'is_sequence', 'capname', 'params'])
```

Comment:
Wish for naming ``TextPart`` instead of ``TextFragment``

term fragment indicates broken parts, but our decoding always occurs in serial order.